src/scene/scene.js

```diff
@@ -214,19 +221,21 @@ class Scene extends EventHandler {
     }
 
     set skybox(value) {
-        this._skyboxCubeMap = value;
-        this._resetSkyboxModel();
-        this.updateShaders = true;
+        if (value !== this._skyboxCubeMap) {
+            this._skyboxCubeMap = value;
+            this._resetSkyboxModel();
+        }
```

Comment:
should updateShaders be set to true if for example we replace current cubemap by no cubemap? (value is null)

Reply:
yes: `_resetSkyboxModel` now sets `this.updateShaders = true;`.